src/main/kotlin/no/nav/syfo/application/syfo/SyfoTilgangsKontrollService.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.syfo.application.syfo
+
+import no.nav.syfo.application.syfo.error.IdentNotFoundException
+import no.nav.syfo.log
+
+class SyfoTilgangsKontrollService(
```

Comment:
Nå har vi to klasser som heter SyfoTilgangskontrollService. Kanskje vi kan bytte navn på denne her til noe annet? :)

Reply:
Ah, ser nå at den ene er service og den andre er client. Kan vi kalle denne her TilgangskontrollService eller noe slikt så det er litt lettere å lese, og så det ikke er så tett kobling til appen syfotilgangskontroll? :)